source/js/main.js

```diff
@@ -38,6 +40,9 @@ let main = {
       csrfToken = document.querySelector(`meta[name="csrf-token"]`);
       csrfToken = csrfToken ? csrfToken.getAttribute(`content`) : false;
 
+      locale = document.querySelector(`meta[property="og:locale"]`);
+      locale = locale ? locale.getAttribute(`content`) : DEFAULT_LOCALE;
```

Comment:
Where is `locale` used in this file?

Reply:
it's not, this provisions it so we can use it for the `/en/` hardcoded link work.